### PR TITLE
login registration with admin user module

### PR DESCRIPTION
**Pull request by sanjeet296**

### CODE REVIEW

File: admin.php [hunk 1]
```diff
@@ -7,13 +7,34 @@
...
+if($_SESSION['isadmin']==0)
+{
+  header('location: user.php');
+  exit();
+}
 
 
 ?>
+ <?php
+$db = mysqli_connect('localhost', 'root', 'java@123', 'sanjeet');
```
Comment (reactexcel) [inline admin.php]: make a seperate file for creating connection with DB and include it.

Comment (niteshiitj) [inline admin.php]: make a separate file for creating connection with DB and include it.

File: admin.php [hunk 2]
```diff
@@ -7,13 +7,34 @@
...
+    $result = mysqli_query($db, $update);
+    header('location: admin.php');
+  }
+}
+
+?> 
 
 <?php
-$db = mysqli_connect('localhost', 'phpmyadmin', 'java@123', 'admin');
+$db = mysqli_connect('localhost', 'root', 'java@123', 'sanjeet');
```
Comment (niteshiitj) [inline admin.php]: same as above, don't create db connection again and again. The connection should be created only once.

File: admin.php [hunk 3]
```diff
@@ -70,15 +98,17 @@
...
 
+ Welcome <?php echo   $_SESSION['fname'].' '.$_SESSION['lname'];?>
+
  <h1 align ="center"> Admin dashboard</h1>
   
    <div class="admin">
   <?php
 
-$db = mysqli_connect('localhost', 'phpmyadmin', 'java@123', 'admin');
+$db = mysqli_connect('localhost', 'root', 'java@123', 'sanjeet');
```
Comment (niteshiitj) [inline admin.php]: same as above.

File: index.php [hunk 4]
```diff
@@ -1,4 +1,8 @@
 <?php
+error_reporting(E_ALL);
```
Comment (niteshiitj) [inline index.php]: for reporting error, you should create a single file, as like i said above for DB connection.

File: poll.php [hunk 5]
```diff
@@ -52,7 +52,7 @@
 </html>
```
Comment (niteshiitj) [inline poll.php]: remove this file from this PR, take a back for it and remove from here.

File: showpoll.php [hunk 6]
```diff
@@ -1,5 +1,9 @@
```
Comment (niteshiitj) [inline showpoll.php]: remove this file from this PR, take a back for it and remove from here.

**Pull request merged**